apps/server/src/tasks/dto/task.input.ts

```diff
@@ -39,8 +38,14 @@ export class TaskInput {
   @IsEnum(Priority)
   priority: string;
 
-  @NullableField(() => ChunkInfoInput)
-  chunkInfo: ChunkInfoInput;
+  @NullableField(() => Interval)
+  minTimeBetweenChunks: Duration;
+
+  @NullableField(() => Interval, { nullable: true })
+  minChunkDuration: Duration;
+
+  @NullableField(() => Interval)
+  maxChunkDuration: Duration;
```

Comment:
No way we're adding that to global task. Const tasks shouldn't have that fields.